examples/binaries/calc-hash/src/lib.rs

```diff
@@ -0,0 +1,60 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+extern crate alloc;
+
+use alloc::vec::Vec;
+use codec::{Decode, Encode};
+use sha2::Digest;
+
+/// Program methods.
+#[derive(Debug, Encode, Decode)]
+pub enum Method {
+    Start(Package),
+    Refuel,
+}
+
+#[derive(Clone, Debug, Encode, Decode)]
+pub struct Package {
+    /// The paths of the calculation
+    pub paths: Vec<[u8; 32]>,
+    /// Expected result.
+    pub expected: [u8; 32],
+}
+
+impl Package {
+    /// verify path
```

Comment:
Please, follow common doc styling
```suggestion
    /// Path verification.
```